src/web/routes/application/address/postcode/os-places.js

```diff
@@ -10,12 +10,18 @@ const standardisePostcode = (postcode) => postcode.toUpperCase().replace(/\s/g,
 const getAddressLookupResults = async (config, postcode) => {
```

Comment:
Does getAddressLookupResults need to by async? Don't we want it to be synchronous?